src/inferenceql/query/plan.cljc

```diff
@@ -268,7 +268,7 @@
     [[:selection child & _]] (aggregator child)
     [[:aggregation aggregation-fn "(" _sym ")"]] (aggregator aggregation-fn)
     [[:aggregation aggregation-fn "(" _distinct _sym ")"]] (aggregator aggregation-fn)
-    [[:aggregation-fn [tag & _]]] (symbol tag)
+    [[:aggregation-fn [tag & _]]] (symbol tag)              ; MUST not munge agg fns
```

Comment:
Hmmm. Based on my current understanding, I'm not sure this is still true. Or at least, it doesn't warrant saying "MUST", rather than "NO NEED". None of the current agg fns contain any unusual characters.

I'll update to say "No need to..."